background.js

```diff
@@ -9,6 +8,18 @@
 
     let isEnabled = true;
     updateIsEnabled();
+    let pyVersion = 3;
```

Comment:
```suggestion
    let pyVersion = '3';
```
I'd suggest using string everywhere, so `===` and `!==` operators can be used.

Reply:
Ah, yup, you're right, I was sloppy with the typing here. I'll convert them all to strings.